experimental/dds/tree2/src/feature-libraries/sequence-field/markListFactory.ts

```diff
@@ -22,7 +22,9 @@ export class MarkListFactory<TNodeChange> {
 	public push(...marks: Mark<TNodeChange>[]): void {
 		for (const item of marks) {
 			if (isNoopMark(item)) {
-				this.pushOffset(item.count);
+				if (item.cellId === undefined) {
```

Comment:
It might be worth adding a brief comment here. Something like "A noop targeting an empty cell can be omitted from the final mark list."

Reply:
Added this comment